README.md

```diff
@@ -38,8 +38,13 @@ Check out [shipkit-changelog](https://github.com/shipkit/shipkit-changelog) plug
 - https://github.com/linkedin/ambry
 - https://github.com/mockito/mockito-scala
 - https://github.com/mockito/mockito-testng
+- https://github.com/mockito/mockito-kotlin
 
-## Usage
+## USAGE
```

Comment:
Nit: Let's use lowercase consistently (or uppercase), e.g. `Usage` here or `CAPITAL` everywhere else for headers

Reply:
Thanks for review! The documentation is now updated with your suggestions.